atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/Trace.common.kt

```diff
@@ -23,6 +23,10 @@ import kotlin.internal.InlineOnly
  * ```
  * trace { "Doing something" }
  * ```
+ * or
+ * ```
+ * trace.append(Thread.currentThread())
+ * ```
```

Comment:
```suggestion
 * or you can do multi-append in a garbage-free manner
 * ```
 * // Before queue.send(element) invocation
 * trace.append("Adding element to the queue", element, Thread.currentThread())
 * ```
```